EasyPost.Tests/ServicesTests/CustomsInfoServiceTest.cs

```diff
@@ -29,6 +29,11 @@ public async Task TestCreate()
             Assert.IsType<CustomsInfo>(customsInfo);
             Assert.StartsWith("cstinfo_", customsInfo.Id);
             Assert.Equal("NOEEI 30.37(a)", customsInfo.EelPfc);
+            Assert.NotEmpty(customsInfo.CustomsItems);
```

Comment:
Nit: This assertion doesn't do anything for us since we'll attempt to iterate the collection on the very next line. Inherently if we can't iterate an object and assert something against it then it was empty (AKA: asserting against individual records will get us this assertion on line 32 for free without needing to add an extra line here)

Prefer to keep code as simple and uncluttered as possible